### ISSUE TITLE
return string response instead of writing always to file system

### ISSUE THREAD

**Issue by parmeet22** [anvilco/spectaql#492]
when using spectaql as a dependency in a node project we do not get a stringified response of the html document generated. This makes it difficult to integrate in an api. 
can we have some feature similar to @asyncapi/generator where we have possbility to get string response instead of getting a HTML file saved to file system always?

**Comment by parmeet22** [comment]
yeah makes sense for embeddable mode or if we have inline or no css and JS. 
This feature would allow me to generate documentation for multiple different introspection.json on the fly without having to always read the output HTML file .

**Comment by newhouse** [comment]
OK, this will be coming soon:
https://github.com/anvilco/spectaql/pull/497

The `run` function will return an object `{html: '<your html here>'}`, which you can do what you please with.

I'll let you know when it's been released.

**Comment by newhouse** [comment]
@parmeet22 this is now in `main` and available in v`1.5.4`
